src/containers/Home/CardBoxList.tsx

```diff
@@ -0,0 +1,62 @@
+import { ListItem } from "@mui/material";
+import List from "@mui/material/List";
+import { CardBox } from "./CardBox";
+
+const cardListArray = [
+    {
+        image: "/test.jpg",
+        name: "Test1",
+        email: "xyz@test.com",
+        phone: "99999999",
+    },
+    {
+        image: "/test.jpg",
+        name: "Test2",
+        email: "xyz@test.com",
+        phone: "09999999",
+    },
+
+    {
+        image: "/test.jpg",
+        name: "Test3",
+        email: "xyz@test.com",
+        phone: "999999999",
+    },
+    {
+        image: "/test.jpg",
+        name: "Test4",
+        email: "xyz@test.com",
+        phone: "999999999",
+    },
+    {
+        image: "/test.jpg",
+        name: "Test5",
+        email: "xyz@test.com",
+        phone: "999999999",
+    },
+];
```

Comment:
Pls, note that this needs to be replaced with API response, once we are done with the API integration.

Reply:
Okay, noted.